packages/std/src/traits.rs

```diff
@@ -1,8 +1,16 @@
-use std::vec::Vec;
-
 use crate::errors::Result;
 use crate::types::{CanonicalAddr, HumanAddr};
 
+#[cfg(feature = "iterator")]
+pub type KVPair = (Vec<u8>, Vec<u8>);
+
+#[cfg(feature = "iterator")]
+#[derive(Copy, Clone)]
+pub enum Sort {
```

Comment:
Since we don't do sorting here (the act of ensuring order where might not be order before), I'd rename this to `Order::Ascending`, `Ascending::Descending`

Reply:
Can you add a comment why you need explicit enum raw values? I guess to pass them over FFI, right?

Reply:
Done